dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/companies/CompanyDataOwners.kt

```diff
@@ -0,0 +1,17 @@
+package org.dataland.datalandbackend.model.companies
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+/**
+ * --- Generic API model ---
+ * DTO for uploading information about company data ownership relations
```

Comment:
DTO?

Reply:
@PaulChr99 could you explain

Reply:
I just used the structure of documentation used in other places in the backend model. DTO is used for DataTravelingObject, isn't it? However, I would not mind changing it, as I didn't pick the term deliberately

Reply:
Ok. Yeah you are right, i didn't saw the other occurence. I will close this now